csm_web/scheduler/factories.py

```diff
@@ -142,13 +149,22 @@ def demoify_user(user, username):
     user.save()
 
 
-def create_demo_accounts():
+def create_demo_account():
     demo_mentor = random.choice(Mentor.objects.all())
-    demoify_user(demo_mentor.user, "demo_mentor")
-    demo_student = random.choice(Student.objects.all())
-    demoify_user(demo_student.user, "demo_student")
-    print("\nDemo accounts have been created with usernames demo_mentor and demo_student")
-    print("The password for these accounts is 'pass', log in at localhost:8000/admin/")
+    second_section = random.choice(Section.objects.filter(
+        course=demo_mentor.section.course).exclude(pk=demo_mentor.section.pk))
+    demo_mentor_2 = second_section.mentor
+    demo_mentor_2.user = demo_mentor.user
+    demo_mentor_2.save()
+    demoify_user(demo_mentor.user, "demo_user")
+    demo_student = random.choice(Student.objects.exclude(section__course=demo_mentor.section.course))
+    demo_student.user = demo_mentor.user
+    demo_student.save()
+    print("""
+    A demo account has been created with username 'demo_mentor' and password 'pass'
+    Log in at localhost:8000/admin/
+    """
```

Comment:
This will print the message with one indent unless you call [textwrap.dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent). Just checking if this indent is your intent.

Reply:
I didn't realize this would have an indent, but I also don't mind it because now that I look at it it does set this off from the progress messages. Will fix only if you want me to.

Reply:
I think the indents fine since it does make it more distinct. Let's leave it as is then.